### ISSUE TITLE
Line color selection window does not provide item names to Narrator

### ISSUE THREAD

**Issue by joshkoon** [microsoft/calculator#955]
**Describe the bug**
When changing an equation style, the colors in the Line Color selection window are not read by Narrator.

**Steps To Reproduce**

1. Go to Graphing mode and add an equation
2. With Narrator on, open the Change Equation Style context menu item from the equation's context menu.
3. Navigate through the colors.
4. Observe Narrator reads out the index of the item in the list, but does not announce the names of the items.

**Expected behavior**
Narrator should announce the names of items in the list.

**Screenshots**

![image](https://user-images.githubusercontent.com/45607479/72635587-fbcd8980-3911-11ea-83e9-65d537098569.png)

**Device and Application Information**
- OS Build: 10.0.19547.0
- Architecture: x64
- Application Version: 10.1912.4.0
- Region: en-US
- Dev Version Installed:

**Additional context**

**Requested Assignment**

I'm just reporting this problem.  I don't want to fix it.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I've seen this issue come in and have gone to tell a human about it.

**Comment by grochocki** [comment]
The color palette was selected using a combination of Windows theme and ink toolbar colors. Our naming should be consistent with these colors. From left-to-right, top-to-bottom, the colors are called:
- Navy Blue
- Seafoam
- Violet
- Green
- Mint Green
- Dark Green
- Charcoal
- Red
- Plum Light
- Magenta
- Yellow Gold
- Orange Bright
- Brown
- Black (or White on dark theme)
